### PR TITLE
added discord icon

### PR DESCRIPTION
**Pull request by usmt-cop**
I have added discord icon and tweaked github icon ui a little bit for a better fit
Let me know if any change needed

### CODE REVIEW

**Comment by michmich112** [comment]
This is great @usmt-cop!
Just one change, can we have the icons stick to the bottom of the page as a footer section?
thx!
Screen Shot 2022-10-05 at 10 47 56 AM

File: src/pages/Landing.svelte [hunk 1]
```diff
@@ -1,108 +1,113 @@
...
+      justify-content: center;
+      align-items: center;
+      gap: 1em;
+      padding: 0px 10px;
+   }
+   .footer {
+      display: flex;
+      justify-content: flex-start;
+      padding: 0rem;
+   }
```
Comment (michmich112) [inline src/pages/Landing.svelte]: Can we make sure that the footer sticks to the bottom of the page?

**Review by michmich112 — CHANGES_REQUESTED** [review]

**Comment by usmt-cop** [comment]
Actually I was doing UI work in my Laptop without considering for potrait view ( laziness 😅)
Let me re-write few custom css

**Comment by usmt-cop** [comment]
I have made the footer to stick in bottom

**Review by michmich112 — APPROVED** [review]

**Comment by michmich112** [comment]
Closes #28

**Pull request merged**